test/integration_test.py

```diff
@@ -1242,19 +1242,19 @@ def _test_managed_access_manifest(self,
         """
         endpoint = config.service_endpoint
 
-        def bundle_uuid(hit: JSON) -> str:
-            return one(hit['bundles'])['bundleUuid']
+        def bundle_uuids(hit: JSON) -> set[str]:
+            return {bundle['bundleUuid'] for bundle in hit['bundles']}
 
-        managed_access_bundles = {
-            bundle_uuid(file)
+        managed_access_bundles = set.union(*(
+            bundle_uuids(file)
             for file in files
             if len(file['sources']) == 1
```

Comment:
For which files is this ever false?

Reply:
https://github.com/DataBiosphere/azul/issues/5330